ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml

```diff
@@ -1,11 +1,10 @@
 tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: origin-v3.11
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
 base_images:
   base:
     cluster: https://api.ci.openshift.org
```

Comment:
I don't believe cluster is strictly needed here. Others don't seem to have it. See https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml as an example

Reply:
Please check now